neptune/new/internal/utils/runningmode.py

```diff
@@ -26,7 +26,8 @@ def in_notebook():
     """Based on: https://stackoverflow.com/a/22424821/1565454"""
     try:
         from IPython import get_ipython
-        if 'IPKernelApp' not in get_ipython().config:  # pragma: no cover
+        ipy = get_ipython()
+        if ipy is None or not hasattr(ipy, 'config') or 'IPKernelApp' not in ipy.config:  # pragma: no cover
```

Comment:
Check that ipy.config is dict :D. Something like or isinstance(ipy.config, dict)